src/read.rs

```diff
@@ -49,6 +50,15 @@ pub fn load_raw_files(folder: &str) -> Vec<SMBiosData> {
     result
 }
 
+/// dumps raw data into a file
```

Comment:
Since this is a file write operation and this file is called read.rs we should change the name of read.rs.  Maybe file_io.rs or just io.rs?

I turned the github discussions feature on and this is a good topic to pick up there - the overall organization of the library.

For now we can either change this file name or leave it as is.

Reply:
Agree, great feedback. I'll rename it.